server/controllers/donations.js

```diff
@@ -61,11 +43,11 @@ const stripeDonation = (req, res, next) => {
     return next(new errors.BadRequest('Stripe Token missing.'));
   }
 
-  if (!amountFloat) {
+  if (!amount) {
     return next(new errors.BadRequest('Payment Amount missing.'));
   }
 
-  if (amountInt < 50) {
+  if (amount < 50) {
```

Comment:
I get it. It'd be better to do, it's a fair amount of work to rename amount everywhere (lots of tables have it, not to mention about 100s of mentions of it in the code - both backend and frontend. Maybe for another weekend...